sdk/appcenter/src/main/java/com/microsoft/appcenter/http/DefaultHttpClientCallTask.java

```diff
@@ -284,17 +284,18 @@ private static InputStream getInputStream(HttpURLConnection urlConnection) throw
                 AppCenterLog.verbose(LOG_TAG, "HTTP response status=" + status + " payload=" + logPayload);
             }
 
+            Map<String, String> responseHeaders = new HashMap<>();
```

Comment:
Add a comment `/* Remove duplicate headers. */` or remove blank line.

Reply:
I was going to push the changes but we got stuck on running code coverage command on my machine. Which didn't work till the end. Pushed the changes now